src/ObjectUtils/include/ObjectUtils/LinuxMap.h

```diff
@@ -26,6 +26,11 @@ ErrorMessageOr<std::vector<orbit_grpc_protos::ModuleInfo>> ReadModules(int32_t p
 ErrorMessageOr<std::vector<orbit_grpc_protos::ModuleInfo>> ParseMaps(
     std::string_view proc_maps_data);
 
+uint64_t SymbolVirtualAddressToAbsoluteAddress(uint64_t symbol_address,
```

Comment:
Should this be called Symbol*Relative*VirtualAddressToAbsoluteAddress to point out the fact that the input address is relative?

Reply:
Ok. I guess it matches `symbol.address()` which is used as argument.